cleo_sdk/CLEO.h

```diff
@@ -460,6 +513,7 @@ void WINAPI CLEO_SetScriptDebugMode(CRunningScript* thread, BOOL enabled);
 
 CRunningScript* WINAPI CLEO_CreateCustomScript(CRunningScript* fromThread, const char* script_name, int label);
 CRunningScript* WINAPI CLEO_GetLastCreatedCustomScript();
+CRunningScript* WINAPI CLEO_GetScriptByName(const char* threadName, BOOL standardScripts, BOOL customScripts, DWORD resultIndex = 0); // returns nullptr if no more scripts found
```

Comment:
CLEO_GetScriptByFileName would help me in adding debugging support for CLEO scripts. Currently it's difficult to find a CLEO script in memory, and script name is not a reliable indicator (multiple CLEO scripts may share the same name). The  file name is unique.